### ISSUE TITLE
Contao 4.9.5 and XAMPP (win10)

### ISSUE THREAD

**Issue by toteph42** [contao/contao#2331]
Contao 4.9.5

**Description**

Not able to install Contao version on XAMPP. Tested CM 1.3.0 and Composer
[composer 4.9.txt](https://github.com/contao/contao/files/5274373/composer.4.9.txt)
[cm 4.9.5.txt](https://github.com/contao/contao/files/5274374/cm.4.9.5.txt)

**Comment by Toflar** [comment]
What happens if you run `vendor/bin/contao-console contao:install-web-dir` manually?

**Comment by toteph42** [comment]
If I run this command in an cmd.exe (with admin rigths) I get no output at all.

**Comment by bezin** [comment]
Can you run `vendor/bin/contao-console.bat contao:install-web-dir`?

**Comment by toteph42** [comment]
`vendor\bin\contao-console.bat contao:install-web-dir -vvv` same result - no messages

**Comment by toteph42** [comment]
Problem is related to PHP crashing. I'll tryi to address the issue to PHP folks. What I did so far:

1. Calling `eventvwr.exe`
2. Checking for a Error
3. Found:
`Name der fehlerhaften Anwendung: php.exe, Version: 7.4.10.0, Zeitstempel: 0x5f4e7d8c`
`Name des fehlerhaften Moduls: VCRUNTIME140.dll, Version: 14.27.29016.0, Zeitstempel: 0x5ee8384d`
`Ausnahmecode: 0xc0000005`
`Fehleroffset: 0x00000000000014a0`
`ID des fehlerhaften Prozesses: 0x23c4`
`Startzeit der fehlerhaften Anwendung: 0x01d69298db157e2e`
`Pfad der fehlerhaften Anwendung: C:\aXa\bin\xampp\php\php.exe`
`Pfad des fehlerhaften Moduls: C:\WINDOWS\SYSTEM32\VCRUNTIME140.dll`
`Berichtskennung: ba789149-36e8-4286-84be-a6c3bf1dee23`
`Vollständiger Name des fehlerhaften Pakets: `
`Anwendungs-ID, die relativ zum fehlerhaften Paket ist: `

**Comment by toteph42** [comment]
Update - I found solution:

Formerly I used XAMPP `xampp-portable-windows-x64-7.4.10-0-VC15.7z`. Now I tried using `xampp-portable-windows-x64-7.4.10-0-VC15-installer.exe`. Nothing changed. 

Then I installed _Visual C++ Redistributable for Visual Studio 2015-2019_ as provided on [https://windows.php.net/download/](url). Nothing changed.

Then I disabled all non-standard extensions in PHP.INI and after running multiple test enabling one PHP extension after the other I found the solution: Disable `php_xdebug-2.9.7-7.4-vc15-x86_64.dll` and CM will work. 

I will address the issue on [https://xdebug.org/](url), but this issue (and the solution) may help other users having problem with Apache and PHP on WIndows 10.
